Lib/test/test_asyncio/test_futures2.py

```diff
@@ -76,21 +76,6 @@ class CFutureTests(FutureTests, unittest.IsolatedAsyncioTestCase):
 class PyFutureTests(FutureTests, unittest.IsolatedAsyncioTestCase):
     cls = tasks._PyTask
 
-class FutureReprTests(unittest.IsolatedAsyncioTestCase):
-
-    async def test_recursive_repr_for_pending_tasks(self):
-        # The call crashes if the guard for recursive call
-        # in base_futures:_future_repr_info is absent
-        # See Also: https://bugs.python.org/issue42183
-
-        async def func():
-            return asyncio.all_tasks()
-
-        # The repr() call should not raise RecursiveError at first.
-        # The check for returned string is not very reliable but
-        # exact comparison for the whole string is even weaker.
-        self.assertIn('...', repr(await asyncio.wait_for(func(), timeout=10)))
```

Comment:
Why did you have to remove this test?

Reply:
Updated, this test now only needs to test that no `RecursionError` is raised as now `asyncio.wait_for` no longer create new task.